effects.cwt

```diff
@@ -317,9 +317,11 @@ alias[effect:add_betrothal] = TODO
 ## scope = character
 ### Adds a�character modifier. Specify duration using�days,�months,�years, or�duration = -1�(non-expiring). Can use optional fields�hidden = yes�and�stacking = yes.
 alias[effect:add_character_modifier] = {
-    name = <event_modifier>
+    enum[modifier_names] = <event_modifier>
     ## cardinality = 0..1
-    duration = -1
+    enum[duration_dmy] = -1
+    ## cardinality = 0..1
+    enum[duration_dmy] = int
```

Comment:
If you're not doing comments, you can just use `int[-1..inf]`.

Also, maybe go through the other `add_x_modifier`s? They probably have similar duration rules.

Reply:
I thought about ``int[-1..inf]`` but would that not allow 0 as well or is 0 an accepted state for ``int`` ?

Reply:
Good point, haven't thought about that.

Reply:
Yea, is there a special way to say -1 AND 1-inf or would Day have to make a special version like ``int[-1..inf-0..0]``

Reply:
I don't think that's worth the effort of adding some more weird syntax in this case.